modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaJAXRSServerCodegen.java

```diff
@@ -95,6 +96,25 @@ public void preprocessSwagger(Swagger swagger) {
             @SuppressWarnings("unchecked")
             List<CodegenOperation> ops = (List<CodegenOperation>) operations.get("operation");
             for ( CodegenOperation operation : ops ) {
+                boolean isMultipartPost = false;
```

Comment:
This addresses an issue where `@FormData` was being used for `multipart/form-data`, causing runtime errors when posting.